libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -325,7 +331,7 @@ void AP_ESC_Telem::send_esc_telemetry_mavlink(uint8_t mav_chan)
         }
 
         bool all_stale = true;
-        for (uint8_t j=0; j<4; j++) {
+        for (uint8_t j = 0; j < 4; j++) {
```

Comment:
This is contrary to most of our code.

Reply:
I see only that there is no consistency regarding spacing in the loops, and if there is a trend, the older code tens to use no spaces, and the newer code tends to use spaces.

- In the (https://ardupilot.org/dev/docs/style-guide.html)[Style Guide], there is nothing about loops, and the only loop example has spaces.
- In this file, loops with spaces are more common (this one was the only loop without spaces).
- `RC_Channel.cpp` has four loops without spaces and one foreach loop with spaces.
- `AP_Scripting.cpp` has three loops without spaces and two loops (one of them foreach) with spaces.
- `AP_Scripting_CANSensor.cpp` has one loop, and it is with spaces.
- `AP_RCProtocol_CRSF.cpp` has five loops, all with spaces.
- `AP_Vehicle.cpp` has three loops with spaces and two "hybrid" loops with no space in the comparison.

Reply:
Okay, will leave this one as is, new version is force-pushed now.